From 4904d384eefcc98bddae124708b1be2e02c70ab5 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 9 Jun 2018 16:27:44 +0300
Subject: [PATCH 6/6] power: Remove unused list utils

* They were added in 6ec1206b246a164acfb0aa7b43af9b04c759063e
  but never used

Change-Id: I0f0d0336cd91715eb8aa4dcb42ae1e449da8603b
---
 list.c | 31 -------------------------------
 list.h |  2 --
 2 files changed, 33 deletions(-)

diff --git a/list.c b/list.c
index 7b9e4d3..1e9ba13 100644
--- a/list.c
+++ b/list.c
@@ -34,16 +34,6 @@
 #include "list.h"
 #include <log/log.h>
 
-int init_list_head(struct list_node *head)
-{
-    if (head == NULL)
-        return -1;
-
-    memset(head, 0, sizeof(*head));
-
-    return 0;
-}
-
 struct list_node *add_list_node(struct list_node *head, void *data)
 {
     /* Create a new list_node. And put 'data' into it. */
@@ -66,11 +56,6 @@ struct list_node *add_list_node(struct list_node *head, void *data)
     return new_node;
 }
 
-int is_list_empty(struct list_node *head)
-{
-    return (head == NULL || head->next == NULL);
-}
-
 /*
  * Delink and de-allocate 'node'.
  */
@@ -107,22 +92,6 @@ int remove_list_node(struct list_node *head, struct list_node *del_node)
     return 0;
 }
 
-void dump_list(struct list_node *head)
-{
-    struct list_node *current_node = head;
-
-    if (head == NULL)
-        return;
-
-    ALOGV("List:\n");
-
-    while ((current_node = current_node->next)) {
-        if (current_node->dump) {
-            current_node->dump(current_node->data);
-        }
-    }
-}
-
 struct list_node *find_node(struct list_node *head, void *comparison_data)
 {
     struct list_node *current_node = head;
diff --git a/list.h b/list.h
index d68c3df..acd02fe 100644
--- a/list.h
+++ b/list.h
@@ -34,8 +34,6 @@ struct list_node {
     void (*dump)(void *data);
 };
 
-int init_list_head(struct list_node *head);
 struct list_node * add_list_node(struct list_node *head, void *data);
 int remove_list_node(struct list_node *head, struct list_node *del_node);
-void dump_list(struct list_node *head);
 struct list_node *find_node(struct list_node *head, void *comparison_data);
-- 
2.17.1

